sdk/src/fee.rs

```diff
@@ -31,16 +31,17 @@ pub struct FeeStructure {
     pub compute_fee_bins: Vec<FeeBin>,
 }
 
-#[derive(Debug, Default, Clone, Eq, PartialEq)]
+#[derive(Debug, Default, Clone, Copy, Eq, PartialEq)]
 pub struct FeeDetails {
     transaction_fee: u64,
     prioritization_fee: u64,
+    remove_rounding_in_fee_calculation: bool,
```

Comment:
nit: `FeeDetails` is public and implements `default`, would making it explicit as `option<bool>` safer?

Reply:
Well you can't set tx or pri fee to a non-zero value via a public method so if the default impl is used, the round feature wouldn't have any effect anyways